lib/rubocop/cop/rspec/single_argument_message_chain.rb

```diff
@@ -22,22 +22,42 @@ class SingleArgumentMessageChain < Cop
           (send _ #{Matchers::MESSAGE_CHAIN.node_pattern_union} $...)
         PATTERN
 
+        def_node_matcher :hash?, '(hash ...)'
+
         def on_send(node)
           message_chain(node) do |(first, *remaining)|
             return if first.to_s.include?('.') || remaining.any?
```

Comment:
I think you could also just match a single argument in the node pattern instead of having to explicitly check if `remaining` is empty.